src/main/java/com/team1816/lib/subsystems/DifferentialDrivetrain.java

```diff
@@ -1,15 +1,15 @@
 package com.team1816.lib.subsystems;
 
 public interface DifferentialDrivetrain extends TrackableDrivetrain {
-    double getLeftVelocityNativeUnits();
-    double getRightVelocityNativeUnits();
+    double getLeftVelocityActual();
```

Comment:
I would consider not using Velocity here how is that different that other velocities?